### PR TITLE
Add an option to preserve existing image tags

### PR DESCRIPTION
**Pull request by zas**
Fix PICARD-257

### CODE REVIEW

File: picard/ui/infodialog.py [hunk 1]
```diff
@@ -88,6 +88,11 @@ def _display_artwork_tab(self):
             if image.width and image.height:
                 infos.append(u"%d x %d" % (image.width, image.height))
             infos.append(image.mimetype)
+            if image in self.obj.metadata.images_to_be_saved_to_tags:
+                if image.preserved:
+                    infos.append("* Will be preserved *")
```
Comment (phw) [inline picard/ui/infodialog.py]: This looks like the strings need to be localized

**Comment by phw** [comment]
Looks good to me :+1: 

Only the strings in the info dialog need to be localized.

**Comment by zas** [comment]
@phw i updated the PR, but it is a lot of more testing, i lack of time for it, and i think it may not work in all situations ... if you can give it  a try and report issues, it will help.

File: picard/ui/options/tags.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ class TagsOptionsPage(OptionsPage):
 
     options = [
         config.BoolOption("setting", "clear_existing_tags", False),
+        config.BoolOption("setting", "preserve_image_tags", False),
```
Comment (phw) [inline picard/ui/options/tags.py]: I'd probably set this to true

**Comment by mineo** [comment]
Hi,

as discussed on [Discourse](https://community.metabrainz.org/t/reducing-the-number-of-open-pull-requests/165542),
I'm going to close this because it has been tagged "Needs more work" and/or
"Update required" and there has been no activity in the last year. I will also
try to keep emails to a minimum and only leave this message on one pull request
per author, so if any other of your pull requests get closed and they match the
criteria, this is why. If new development happens on this pull request, please
leave a comment so we can reopen it!

**Comment by hashhar** [comment]
I'd like to pick up where others left off. Looking at the code the only thing required now looks like testing?

**Comment by phw** [comment]
@zas Any chance to revive this?

**Comment by zas** [comment]
> @zas Any chance to revive this?

Yes, but it was messy, i'd rather start by a cover art stuff cleanup first (help welcome).

**Pull request closed without merging**